Test Results - Average Loss: 0.5151, Average Dice Score: 0.7595100%[Finetuning] Epoch: 1 Avg Loss: 0.3152[Validation] Avg Loss: 0.3181, Avg Dice Score: 0.8984100%[Finetuning] Epoch: 2 Avg Loss: 0.3067[Validation] Avg Loss: 0.3140, Avg Dice Score: 0.9013100%[Finetuning] Epoch: 3 Avg Loss: 0.3015[Validation] Avg Loss: 0.3211, Avg Dice Score: 0.8932100%[Finetuning] Epoch: 4 Avg Loss: 0.3006[Validation] Avg Loss: 0.3235, Avg Dice Score: 0.8864100%[Finetuning] Epoch: 5 Avg Loss: 0.2949[Validation] Avg Loss: 0.3147, Avg Dice Score: 0.8988100%[Finetuning] Epoch: 6 Avg Loss: 0.2934[Validation] Avg Loss: 0.3237, Avg Dice Score: 0.8825100%[Finetuning] Epoch: 7 Avg Loss: 0.2921[Validation] Avg Loss: 0.3148, Avg Dice Score: 0.8976100%[Finetuning] Epoch: 8 Avg Loss: 0.2909[Validation] Avg Loss: 0.3148, Avg Dice Score: 0.8959100%[Finetuning] Epoch: 9 Avg Loss: 0.2878[Validation] Avg Loss: 0.3160, Avg Dice Score: 0.8957 100%||[Finetuning] Epoch: 10 Avg Loss: 0.2870[Validation] Avg Loss: 0.3173, Avg Dice Score: 0.8935 /home/mikhelson/Med143/143 [00:13<00:00, 10.64it/s]143/143 [00:13<00:00, 10.75it/s]143/143 [00:13<00:00, 10.73it/s]143/143 [00:13<00:00, 10.63it/s]143/143 [00:13<00:00, 10.52it/s]143/143 [00:13<00:00, 10.79it/s]143/143 [00:13<00:00, 10.70it/s]143/143 [00:13<00:00, 10.80it/s]143/143 [00:13<00:00, 10.57it/s]143/143 [00:13<00:00, 10.73it/s]ImSeg-Lab24/CLUE/lib/python3.8/site-packages/lightning/pytorch/core/module.py:447: You are trying to `self.log()` but the 'self.trainer reference is not registered on the model yet. This is most likely because the model hasn't been passed to the 'Train Test Results - Average Loss: 0.6640, Average Dice Score: 0.7035er'